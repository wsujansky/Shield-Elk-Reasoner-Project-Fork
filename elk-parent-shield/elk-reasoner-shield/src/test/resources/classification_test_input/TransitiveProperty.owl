Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#transitive_property>
TransitiveObjectProperty(:T)
SubClassOf(:A ObjectSomeValuesFrom(:T :B))
EquivalentClasses(:B ObjectSomeValuesFrom(:T :C))
)