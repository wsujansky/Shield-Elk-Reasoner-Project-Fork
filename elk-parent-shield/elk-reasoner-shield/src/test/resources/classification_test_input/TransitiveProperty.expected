Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#transitive_property#taxonomy>
Declaration(Class(:A))
Declaration(Class(:B))
Declaration(Class(:C))
SubClassOf(:A :B)
)